### DISCUSSION TITLE
Issue with TempFanTower Post Processing GCode

### DISCUSSION THREAD

**Discussion by eblieb** [5axes/Calibration-Shapes#52] [General]
For some reason TempFanTower has the change layer automatically set at layer 52 which doesnt seem correct based on my slicing.

**Comment by 5axes** [comment]
52 is the default value ...  This value must be change according to your settings , if you use a different **Layer height** or **Initial layer height** or even your own Tower Geometry  this parameter .is not automaticaly set to the right value

The provided script are just an "Help" to creat a Tower. https://github.com/5axes/Calibration-Shapes/wiki/Customize-script

**Reply by eblieb** [reply]
That makes a lot of sense. I forgot about layer height changes. Is there anyway to make it where you can save your own setting there

**Reply by 5axes** [reply]
https://github.com/5axes/Calibration-Shapes/issues/45
